drivers/tapdisk-nbdserver.h

```diff
@@ -98,5 +100,28 @@ int tapdisk_nbdserver_listen_unix(td_nbdserver_t *server);
 void tapdisk_nbdserver_free(td_nbdserver_t *);
 void tapdisk_nbdserver_pause(td_nbdserver_t *);
 int tapdisk_nbdserver_unpause(td_nbdserver_t *);
+void tapdisk_nbdserver_clientcb(event_id_t id, char mode, void *data);
```

Comment:
It is not part of the fix nor the code re-factoring but.. given you are in documentation mode and this
function is used later on in the tests, do you mind writing a short doc? Pleeeeeeease :)